src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/IncrementingEventCounter.cs

```diff
@@ -56,6 +57,9 @@ public void Increment(double increment = 1)
 
         public override string ToString() => $"IncrementingEventCounter '{Name}' Increment {_increment}";
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "IncrementingCounterPayload type has been trimmer safe")]
```

Comment:
Similar justification message here as above - explain that the DynamicDependency makes this safe.